tools/add_triage_signature.py

```diff
@@ -1057,6 +1057,57 @@ def _update_ticket(self, url, issue_key, should_update=False):
             self._update_triaging_ticket(issue_key, report, should_update=should_update)
 
 
+class WrongBootOrderSignature(ErrorSignature):
+    """
+    """
+
+    ERROR_PATTERN = "a manual booting from installation disk"
+    EVENT_PATTERN = "please boot the host"
+
+
+    def __init__(self, jira_client):
+        super().__init__(jira_client, signature_label="wrong_boot_order",
+                         comment_identifying_string="h1. Wrong boot order")
+
+    def _update_ticket(self, url, issue_key, should_update=False):
+        url = self._logs_url_to_api(url)
+
+        md = get_metadata_json(url)
+
+        cluster = md['cluster']
+        status_info = cluster['status_info']
+
+        if self.ERROR_PATTERN not in status_info:
+            return
+
+        cluster_id = cluster['id']
+        report = ""
+        events = get_events_json(cluster_url=url, cluster_id=cluster_id)
+        reboot_events_by_host = defaultdict(list)
+        for event in events:
+            if self.EVENT_PATTERN in event["message"]:
+                reboot_events_by_host[event["host_id"]].append((event))
```

Comment:
the parenthesis are redundant, maybe you meant this?
```suggestion
                reboot_events_by_host[event["host_id"]].append((event,))
```